File: QViz.NET/Periscope.QViz/JSON/Tag.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model Class for Tag Details
	/// </summary>
	public class Tag
	{
		/// <summary>
		/// Identifier of the Tag
		/// </summary>
		public string tagId;
		
		/// <summary>
		/// Name of the Tag
		/// </summary>
		public string tagName;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public Tag()
		{
			this.tagId = "";
			this.tagName = "";
		}

		/// <summary>
		/// Create a new Tag with the provided values
		/// </summary>
		/// <param name="id">Identifier of the Tag</param>
		/// <param name="name">Name of the Tag</param>
		public Tag(string id, string name)
		{
			this.tagId = id;
			this.tagName = name;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/Tag.java
package com.periscope.qviz.json;

/**
 * JSON Model Class for Tag Details
 */
public class Tag {

	/**
	 * Identifier of the Tag
	 */
	public String tagId;

	/**
	 * Name of the Tag
	 */
	public String tagName;

	/**
	 * Default Constructor
	 */
	public Tag() {
		this.tagId = "";
		this.tagName = "";
	}

	/**
	 * Create a new Tag with the provided values
	 *
	 * @param id   Identifier of the Tag
	 * @param name Name of the Tag
	 */
	public Tag(String id, String name) {
		this.tagId = id;
		this.tagName = name;
	}

}
